apps/connector/sources/store.py

```diff
@@ -14,17 +14,24 @@
 import random
 
 
+sources_list = [
+    relief_web.ReliefWeb,
+    atom_feed.AtomFeed,
+    rss_feed.RssFeed,
+    emm.EMM,
+    acaps_briefing_notes.AcapsBriefingNotes,
+    unhcr_portal.UNHCRPortal,
+    pdna.PDNA,
+    research_center.ResearchResourceCenter,
+    wpf.WorldFoodProgramme,
+    humanitarian_response.HumanitarianResponse,
+]
+
+assert len(set([source.key for source in sources_list])) == len(sources_list), 'Duplicate key found for connectors'
```

Comment:
This assertion is for hardcoded values to avoid duplicate keys for each connector source. 

The reason to be strict is that it needs to be strict :stuck_out_tongue:.